Add sign up link

EvgeniiMaiorov opened the pull request:
https://trello.com/c/VXk1pItU/17-add-sign-up-link

In order to redirect users to the registration page we have add a sign up link

Add react-router-dom lib
Create SignUp page
Add link to SignUp page
Remove turbolinks for the application to work correctly

Commit:
Add sign up link

In order to redirect users to the registration page we have add a sign up link

Add react-router-dom lib
Create SignUp page
Add link to SignUp page
Remove turbolinks for the application to work correctly

anevsevra commented on app/javascript/packs/App.jsx:
```diff
@@ -14,10 +16,17 @@ html, body {
-    <>
-      <MainPage />
+    <Router>
       <Global />
```
Move `Global` component to upper level. It shouldn't be in `Router` component.

anevsevra commented on app/javascript/packs/App.jsx:
```diff
@@ -14,10 +16,17 @@ html, body {
   ReactDOM.render(
-    <>
-      <MainPage />
+    <Router>
```
It makes sense to move router to separate standalone component.

Commit:
Move router in standalone component

Move router in standalone component
Remove global from router component to upper level
